src/thor/dijkstras.cc

```diff
@@ -498,7 +498,8 @@ void Dijkstras::ExpandForwardMultiModal(GraphReader& graphreader,
 
     // Reset cost and walking distance
     Cost newcost = pred.cost();
-    uint32_t walking_distance = pred.path_distance();
+    uint32_t walking_distance = pred.walking_distance();
+    uint32_t path_distance = pred.path_distance();
```

Comment:
separate walking & path_distance so we can report the right full distance in the summary object

Reply:
unfortunately also means it has to be a new field in the MMEdgeLabel